systemd/ua-update-contract.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=Ubuntu Advantage auto attach
```

Sorry cut-n-paste error of mine here.
```suggestion
Description=Ubuntu Advantage update contract
```